web/client-api/src/main/java/io/deephaven/web/client/api/widget/JsWidgetExportedObject.java

```diff
@@ -29,13 +34,46 @@ public class JsWidgetExportedObject implements ServerObject {
 
     private final TypedTicket ticket;
 
+    private final JsLazy<Promise<?>> fetched;
+
     public JsWidgetExportedObject(WorkerConnection connection, TypedTicket ticket) {
         this.connection = connection;
         this.ticket = ticket;
+        this.fetched = JsLazy.of(() -> {
+            if (getType() == null) {
+                return Promise.reject("Exported object has no type, can't be fetched");
```

Comment:
Is this a case that actually happens?

Reply:
it sure can - make some python object that doesnt have a plugin, and send it from the server using `stream.on_data(..., [my_obj])`. The object will be exported, the client will have a reference to it (but can't `fetch()` it), can pass that back to the server via `dh.Widget.sendMessage()`.

Reply:
Ah yea true. Client will want to close those, but not fetch those... If you do try and fetch it, it'll reject, but then you won't be able to close it. Release the ticket here instead, since they already screwed up trying to fetch something they shouldn't?

Reply:
I've locally added a guard to prevent fetching (and throw an error in that case), but close() can still be done later.